docs/pages/docs/tile/tile-group.mdx

```diff
@@ -52,9 +52,74 @@ FTileGroup(
 );
 ```
 
+### `FTileGroup.merge(...)`
+
+```dart
+FTileGroup.merge(
+  label: const Text('Settings'),
+  semanticLabel: 'Settings',
+  divider: FTileDivider.indented,
+  children: [],
+);
+```
+
 ## Examples
 
-### Full Divider
+### Merge Multiple Groups
+
```

Comment:
I think it could be nice to leave a note about when to use the merge and what's it is made for.